inst/plumber/17-arguments/plumber.R

```diff
@@ -0,0 +1,111 @@
+#* Works with query string arguments
+#* @serializer print
+#* @post /bad-practice/<a>/<b>
+function(a, b) {
+  list(a = a, b = b)
+}
+
+
+
+#* Can have conflicting path, query, and post body arguments
```

Comment:
```suggestion
#* Since URI paths, query params, and post body arguments can have conflicting names, it's better practice to access arguments via the request object
```